hypha/templates/includes/login_button.html

```diff
@@ -2,23 +2,68 @@
-            {% heroicon_micro "user" class="inline align-text-bottom w-4 h-4 me-1" aria_hidden=true %}
-
-            {% blocktrans %}My {{ ORG_SHORT_NAME }}{% endblocktrans %}
-        </a>
+        <div x-data="{ show: false }" class="relative flex">
+            <button x-on:click="show = ! show" class="button--narrow font-bold text-center flex items-center justify-center px-2 hover:bg-black hover:text-white active:bg-black active:text-white focus:bg-black focus:text-white" type="button">
+                {% heroicon_micro "user-circle" class="inline align-text-bottom w-8 h-8 me-1" aria_hidden=true %}
+                {% blocktrans %}My {{ ORG_SHORT_NAME }}{% endblocktrans %}
+            </button>
+            <div x-show="show" x-transition @click.outside="show = false" class="min-w-36 absolute block bg-white shadow-xl z-20 border-y rounded-sm end-0 top-10">
+                <a href="{% url 'users:account' %}" title="Goto your account" class="pr-4 py-3 block text-black pl-3
+                                                                                     focus-visible:outline-dashed outline-1 transition-colors border-r
+                                                                                     hover:bg-slate-100 hover:text-dark-blue hover:font-semibold">
+                    {% trans 'My account' %}</a>
+                {% if request.user.is_apply_staff %}
+                    <a href="{% url 'activity:notifications' %}" title="Check latest notifications" class="pr-4 py-3 block text-black pl-3
+                                                                                                           focus-visible:outline-dashed outline-1 transition-colors border-r
+                                                                                                           hover:bg-slate-100 hover:text-dark-blue hover:font-semibold" >
+                        {% trans "Activity feed" %}</a>
+                    <a href="{% url 'apply:submissions:flagged' %}" title="Goto your flagged submissions" class="pr-4 py-3 block text-black pl-3
+                                                                                                                 focus-visible:outline-dashed outline-1 transition-colors border-r
+                                                                                                                 hover:bg-slate-100 hover:text-dark-blue hover:font-semibold">
+                        {% trans "My flagged" %}</a>
+                    <a href="{% url 'wagtailadmin_home' %}" title="Goto wagtail admin" class="pr-4 py-3 block text-black pl-3
+                                                                                              focus-visible:outline-dashed outline-1 transition-colors border-r
+                                                                                              hover:bg-slate-100 hover:text-dark-blue hover:font-semibold">
+                        {% trans "Wagtail admin" %}</a>
```

Comment:
```suggestion
                        {% heroicon_mini "wrench-screwdriver" size=20 class="mr-1 inline align-text-bottom" %} {% trans "Admin" %}</a>
```

what are the thoughts on swapping this to be more inline with #3922?

Reply:
I just copied the line from #3922, the tailwind may need some tweaking